session.go

```diff
@@ -517,6 +518,19 @@ func (q *Query) Bind(v ...interface{}) *Query {
 	return q
 }
 
+// SerialConsistency sets the consistencyc level for the
+// serial phase of conditional updates. That consitency can only be
+// either SERIAL or LOCAL_SERIAL and if not present, it defaults to
+// SERIAL. This option will be ignored for anything else that a
+// conditional update/insert.
+func (q *Query) SerialConsistency(cons Consistency) *Query {
```

Comment:
Is it such a great idea to panic here? I was under the impression that we refrained from panicking in this library.